modules/project-docs/pages/third-party-integrations.adoc

```diff
@@ -0,0 +1,30 @@
+= Integrations & Ecosystem
+:description: The Couchbase Scala SDK is often used with unofficial and third party tools and applications to integrate into broader language and platform ecosystems, and across data lakes in heterogeneous environments.
+:navtitle: Integrations
+:page-topic-type: project-doc
+:page-aliases: 
+
+[abstract]
+{description}
+
+include::7.0@sdk:shared:partial$integrations.adoc[tag=intro]
+
+
+include::7.0 @sdk:shared:partial$integrations.adoc[tag=official]
```

Comment:
Extra space here after `7.0`

Reply:
Fixed - you should have editing rights on these patch branches, btw - I've been able to fix typos in them even when they're in someone else's fork  :+1: